app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -246,25 +251,24 @@ export const RuleGroup = ({ pluginName, typedRules, notifications, isPluginEnabl
               </Toggle>
             </div>
           </FieldElement>
-          {isUpdateSettingAvailable &&
-            (!isPluginEnabled ? (
-              <div className={cx('disabled-plugin')}>
-                <p>
-                  <span className={cx('capitalized')}>{pluginName}</span>{' '}
-                  {formatMessage(messages.disabledPlugin, { pluginName })}
-                </p>
-                <RuleItemDisabledTooltip
-                  className={cx('info-tooltip')}
-                  tooltipContent={formatMessage(messages.disabledContactInfo)}
-                >
-                  <i className={cx('icon', 'about-icon')}>{Parser(AboutIcon)}</i>
-                </RuleItemDisabledTooltip>
-              </div>
-            ) : (
-              pluginName === EMAIL &&
-              !isEmailIntegrationAvailable && (
-                <div className={cx('integrate-configurations')}>
-                  <p>{formatMessage(messages.notConfiguredIntegration)}</p>
+          {isDisabledTooltipActivationRequired ? (
```

Comment:
Can be used the `isActivationRequired`?

Reply:
In this case, there is a need to include 'isPluginEnabled' in the condition.